trunk/lib/utils/general-usage-functions.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
-function yotpo_get_arr_value(array $object = [], string $key, $default = '') {
+function yotpo_get_arr_value(array $object = [], string $key = '', $default = '') {
```

Comment:
what's the reason for default value? is this function used anywhere without key? if so - why?

Reply:
![image](https://github.com/YotpoLtd/woocommerce-plugin-yotpo-reviews/assets/19251804/2da9ea5a-7c96-4654-be20-e83f6db6f945)

Reply:
yes, I saw that but deprecation was in PHP8 so why this error showed up now? unless someone updated php